src/System.Runtime/tests/System/Attributes.netstandard1.7.cs

```diff
@@ -137,7 +137,11 @@ public static class GetCustomAttribute
         public static void customAttributeCount()
         {
             List<CustomAttributeData> customAttributes =  typeof(GetCustomAttribute).Module.CustomAttributes.ToList();
-            Assert.Equal(3, customAttributes.Count);
+            // [System.Security.UnverifiableCodeAttribute()]
+            // [TestAttributes.FooAttribute()]
+            // [TestAttributes.ComplicatedAttribute((Int32)1, Stuff = 2)]
+            // [System.Diagnostics.DebuggableAttribute((Boolean)True, (Boolean)False)]
```

Comment:
I think I'll leave it as is for now. I had logged it to figure out why it changed to 4. It is the UnverifiableCodeAttribute. I do not understand why exposing it on the reference assembly makes it appear on the test fixture assembly.